binary-search/download_imdb.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+"""
+Fetch and parse people names from the IMDb.
+
+Usage:
+$ python download_imdb.py
+"""
+
+import csv
+import gzip
+import shutil
+import tempfile
+import urllib.request
+
+
+def main():
+    """Script entry point."""
+
+    print("Fetching data from IMDb...")
```

Comment:
On a slow connection, this might take a while. You might want to call that out in the `print` statement, but if it's in the article text, then no worries.